oak_functions/examples/abitest/client/src/main.rs

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+//! Oak Functions ABI test client.
```

Comment:
Why is the entire abitest implemented as an example instead of a test? Is it possible to make it a test instead?

Reply:
Currently submitting it as an example so it can be refactored to be a test as part of: https://github.com/project-oak/oak/issues/1963